packages/vue/config/storybook/home/home.js

```diff
@@ -16,32 +16,32 @@ storiesOf(" |Introduction", module)
     <div class="intro">
       <h1>StorefrontUI - Customization-first component library</h1>
       <p>
-        Welcome in our storybook!
+        Welcome to our storybook!
         You can use it to browse and play with StorefrontUI components.
-        On the left panel you will find every component that is currently available in the library. 
+        On the left panel, you will find every component that is currently available in the library. 
         You can play with props and CSS modifiers of each component from <b>Knobs menu</b>. 
-        Components marked as <i>WIP</i> will much likely change and they;re not safe to use.
-        On the <b>notes</b> menu you will find additional information about component such as available SCSS variables.
+        Components marked as <i>WIP</i> will much likely change and they're not safe to use.
```

Comment:
`Components marked as <i>WIP</i> will likely be changed, hence it's not safe to use them.`